errors.json

```diff
@@ -353,8 +318,8 @@
 		]
 	},
 	"TokenInvalid": {
-		"description": "Authorization token invalid or expired.",
-		"message": "Session has expired.",
+		"description": "Authorization token is invalid or has expired.",
+		"message": "Session has expired. Please authenticate again.",
```

Comment:
just nitpicking here:
- I think the description is a better message than the message here (expiry is not the only reason a token can be invalid). Also, I wouldn't add "Please authenticate again." 
- Also change error code to "AuthenticationTokenInvalid" (or "AuthTokenInvalid")?

Reply:
> * I think the description is a better message than the message here (expiry is not the only reason a token can be invalid).

Well, from a user perspective it's probably hard to understand what invalid means. But I changed it to `Authorization token has expired or is invalid. Please authenticate again.` so that expires is the first reason to read.

> Also, I wouldn't add "Please authenticate again."

I think as a user I would want to know what to do next. Why should that part be a problem? Maybe it's easier to say "log in" though.

> * Also change error code to "AuthenticationTokenInvalid" (or "AuthTokenInvalid")?

As it's really mostly for internal use (the error code name), Ieaving as is for now to not force too many changes.